redaxo/src/core/fragments/core/Component/Range.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Redaxo\Core\Fragment\Component\Range;
+use Redaxo\Core\Fragment\Fragment;
+
+/** @var Range $this */
+?>
+
+<sl-range <?= $this->attributes->with([
+    'name' => $this->name,
+    'value' => $this->value,
+    'label' => is_string($this->label) ? $this->label : null,
+    'help-text' => is_string($this->notice) ? $this->notice : null,
+    'disabled' => $this->disabled,
+    'min' => $this->min,
+    'max' => $this->max,
+    'step' => $this->step,
+    'tooltip' => $this->tooltipPlacement,
+])->toString() ?>>
+    <?= $this->label instanceof Fragment ? Fragment::slot($this->label, 'label') : '' ?>
```

Comment:
lt. type in https://github.com/redaxo/redaxo/pull/5811/files#diff-da77e244ab1fc2084d520d52839eb123c0b40c5cd5bece55a7bb17adc8336bedR17 kann `label` auch ein string sein... müsste man den dann hier auch ausgeben?